src/AppBundle/Repository/EventRepository.php

```diff
@@ -264,19 +264,17 @@ public function getRevisions(Event $event, ?int $offset = 0, ?int $limit = 50, b
         // Check cache and return if it exists, unless the Event was recently updated,
         // in which case we'll want to invalidate the cache.
         $shouldUseCache = null !== $event->getUpdated() &&
-            (int)$event->getUpdated()->format('U') < time() - $cacheDuration &&
-            null !== $this->getRedisConnection();
+            (int)$event->getUpdated()->format('U') < time() - $cacheDuration;
         $cacheKey = $this->getCacheKey(func_get_args(), 'revisions');
-        if ($shouldUseCache && $this->getRedisConnection()->contains($cacheKey)) {
-            return $this->getRedisConnection()->fetch($cacheKey);
+        if ($shouldUseCache && $this->cache->hasItem($cacheKey)) {
+            return $this->cache->getItem($cacheKey)->get();
```

Comment:
According to [this doc](https://api.symfony.com/4.1/Symfony/Component/Cache/Simple/ApcuCache.html), there is a simpler `ApcuCache`. Looks like we could use the simpler `get($key...)` and `set($key...)`. Are there performance differences between that and the full [`ApcuAdapter`](https://api.symfony.com/4.1/Symfony/Component/Cache/Adapter/ApcuAdapter.html) which does provide `getItem()`?

Reply:
It's actually using https://symfony.com/doc/current/components/cache/cache_pools.html, the PSR-6 implementation that allows you to change the caching backend without having to update any code.